src/main/java/trello/ui/pages/BoardPage.java

```diff
@@ -61,6 +62,12 @@
     @FindBy(className = "js-add-a-card")
     private WebElement addACard;
 
+    @FindBy(className = "js-card-details")
+    private List<WebElement> cards;
+
+    @FindBy(className = "js-open-add-list")
+    private WebElement addAnotherLits;
```

Comment:
Please fix the typo in the variable name

Reply:
Solved